### DISCUSSION TITLE
can you guys add defroum extention

### DISCUSSION THREAD

**Discussion by hadoizz** [Meonako/webui-api#9] [Q&A]
hey can u guys guide me for deforum extnetion api ?

**Comment by Lebenoa** [comment]
I'm afraid I can't help you with that but maybe others can.

First of all, I don't really use Web UI (or something along this way) that much anymore. I still use it like **at least** once per 2 days tho.

Second, I don't use deforum and I don't even know what that is until camenduru guys install it as default in his colab repo but I still had no idea what is it since I'm so dumb.

Maybe others can help you.
